src/components/Select/Select.test.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { render, screen, axe } from 'testUtils';
+
+import Select from './';
+import Label from '../Label';
+
+test('renders correctly', () => {
+  render(<Select>Options</Select>);
+
+  expect(screen.getByText('Options')).toBeInTheDocument();
```

Comment:
```suggestion
  expect(screen.getByRole('combobox')).toBeInTheDocument();
```
Check the list of roles here - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#roles